pyface/__init__.py

```diff
@@ -19,7 +19,7 @@
 
 
 __requires__ = [
-    'importlib-metadata; python_version<"3.8"',
+    'importlib-metadata>=3.9.0; python_version<"3.8"',
```

Comment:
This is a very nitpicky suggestion but it looks like the actual minimum required version is 3.6.0 - not 3.9.0. The `select` interface was introduced in version 3.6.0. Reference - https://importlib-metadata.readthedocs.io/en/latest/history.html#v3-6-0

```suggestion
    'importlib-metadata>=3.6.0; python_version<"3.8"',
```